metpy/plots/declarative.py

```diff
@@ -838,6 +840,7 @@ def _build(self):
         self.handle = self.parent.ax.imshow(imdata, extent=extents, origin=origin,
                                             cmap=self._cmap_obj, norm=self._norm_obj,
                                             transform=imdata.metpy.cartopy_crs)
+        plt.colorbar(self.handle, orientation=self.colorbar, pad=0, aspect=50)
```

Comment:
With the change above, can you wrap this in an: `if self.colorbar is not None:`?

Reply:
Done.